memcache.py

```diff
@@ -507,7 +504,7 @@ def delete_multi(self, keys, time=None, key_prefix='', noreply=False):
                 write(cmd)
             try:
                 server.send_cmds(b''.join(bigcmd))
-            except socket.error as msg:
+            except OSError as msg:
```

Comment:
Can you point me at a recommendation to replace socket.error with OSError?  Looks like in the 3.12 source code they document socket.error as an alias for OSError, so this is correct, but I'm reluctant to change it without seeing where the recommendation comes from.

Reply:
Scratch that, I found where it is deprecated.  Good spot!